src/ctia/link/routes.clj

```diff
@@ -0,0 +1,75 @@
+(ns ctia.link.routes
```

Comment:
Why not add this route to the `ctia.entity.incident` or `ctia.entity.relationship` ns? For me a link between a casebook and a incident is a `Relationship`.